Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs

```diff
@@ -19,45 +19,42 @@ public class DestructibleComponent : Component, IOnDamageBehavior
         /// <inheritdoc />
         public override string Name => "Destructible";
 
-        /// <inheritdoc />
-        public override uint? NetID => ContentNetIDs.DESTRUCTIBLE;
-
         /// <summary>
         /// Damage threshold calculated from the values
         /// given in the prototype declaration.
         /// </summary>
         [ViewVariables]
         public DamageThreshold Threshold { get; private set; }
 
+        public DamageType damageType = DamageType.Total;
+        public int damageValue = 0;
+        public string spawnOnDestroy = "SteelSheet";
```

Comment:
Ideally this would probably be a list of prototypes, not just one single prototype.
but for now I guess it doesn't matter.